README.md

```diff
@@ -139,7 +139,39 @@ db.upsert("Collection1", user1, "Users");
 db.delete("Collection1", user1,id, "Users");
 ```
 
+### Batch Operation
+```java
+var db = new Cosmos(System.getenv("YOUR_CONNECTION_STRING")).getDatabase("Database1");
+
+// Create
+db.batchCreate("Collection1", List.of(new User("id011", "Tom", "Banks")), "Users");
+
+// Upsert
+db.batchUpsert("Collection1", List.of(user1), "Users");
+
+// Delete
+db.batchDelete("Collection1", List.of(user1), "Users");
+// or
+db.batchDelete("Collection1", List.of(user1.id), "Users");
+
+```
 
+### Bulk Operation
```

Comment:
Please write the difference between batch operation and bulk operation in the README🙏